content.js

```diff
@@ -113,12 +125,33 @@ function parseGamedataApiXml(str) {
     return game
 }
 
-function findBGGGameId(gameName) {
+function findExactBGGGameId(gameName) {
     query = "https://www.boardgamegeek.com/xmlapi2/search?query=" + String(gameName).replaceAll(' ', '+').replaceAll(':', '+') + "&type=boardgame&exact=1";
     return (
         fetch(query)
             .then(searchResponse => searchResponse.text())
-            .then(searchText => parseSearchApiXml(searchText))
+            .then(searchText => parseExactSearchApiXml(searchText))
+    )
+}
+
+function getGameReleaseYearFromBGA() {
+    let game_info_col = document.getElementsByClassName("col-md-4 game_infos_first_col")[0];
```

Comment:
I don't think the col-md-4 is needed here. 

```suggestion
    let game_info_col = document.getElementsByClassName("game_infos_first_col")[0];
```

Reply:
Does it do partial matches?

Reply:
Yes. In this example, the field has 2 different classNames, you could use either one to query using `getElementsByClassName`. But in this case, the real important class name is the second one.

Reply:
Great, thanks for the explanation :)